tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/es/ElasticsearchFinishedImportingIT.java

```diff
@@ -181,7 +183,9 @@ final var record = generateRecord(ValueType.PROCESS_INSTANCE, "8.7.0", 1);
     EXPORTER.export(partitionTwoRecord2);
     tasklistEsClient.indices().refresh(new RefreshRequest("*"), RequestOptions.DEFAULT);
 
-    for (int i = 0; i <= RecordsReaderHolder.MINIMUM_EMPTY_BATCHES_FOR_COMPLETED_READER; i++) {
+    for (int i = 0;
+        i <= tasklistProperties.getImporter().getCompletedReaderMinEmptyBatches();
```

Comment:
🔧 Maybe we refactor this and store the value in a field, and reuse the field. To simply reduce some boiler plate.